risc0/zkvm/compat/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "risc0-zkvm-compat-tests"
+version = "0.1.0"
+edition = "2021"
```

Comment:
I think we want to set release = "false" here to prevent this crate from being published

Reply:
Good call